MissingLetter.kt

```diff
@@ -0,0 +1,10 @@
+class MissingLetter {
+    fun missingLetter(array: Array<Char>): Char {
+        var executableLetter:Char = array[0];
+        for(letter:Char in array) {
+            if(letter != executableLetter) break;
+            executableLetter++;

```

Comment:
`executableLetter` is an odd choice of a name, I think something like `expectedLetter`